index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Divyansh Singh</title>
+   <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <div>
+    <header>
+    <nav class="header">
+       <ul class="list">
+           <li class="item" id="name"><a href="#">Divyansh Singh</a></li>
+        </ul>
+        <img id="header-image" src="resources/Union.png" alt="menu">
+    </nav>
+
+    </header>
+    <section>
+        <h1 class="heading" id="main">Divyansh Singh</h1>
+    <h2 class="heading" id="job">Software Developer</h2>
+        <p class="para">
+            I can help you build a product , feature or website.
+             Look through some of my work and experience!If you 
+             like what you see and have a project you need coded,
+            don’t hestiate to contact me.
+            
+        </p>
+       
+    </section>
+    <img src="resources/Devider.png" class="divider" alt="divider">
+    <section class="spec">
+       
+        <ul>
+            <li class="list-heading">Language</li>
+          
+            <li><a href="#">HTML</a></li>
+            
+            <li><a href="#">CSS</a></li>
+         
+            <li><a href="#">Javascript</a></li>
+      
+            <li><a href="#">Ruby</a></li>
+        </ul>
+       
+       
+        <ul>
+            <li class="list-heading">Frameworks</li>
+     
+      
+            <li><a href="#">Bootstrap</a></li>
+      
+            <li><a href="#">Ruby On Rails</a></li>
+ 
+            <li><a href="#">RSPec</a></li>
+
+            <li><a href="#">CapyBara</a></li>
+      
+            <li><a href="#">Selenium</a></li>
+        </ul>
+        
+        <ul class="end">
+            <li class="list-heading">Skills</li>
+            
+        <li><a href="#">Codekit</a></li>
+    
+        <li><a href="#">Github</a></li>
+
+        <li><a href="#">Codepen</a></li>
+
+        <li><a href="#">Gitlab</a></li>
+
+        <li><a href="#">Terminal</a></li>
+        </ul>
+       
+    </section>
+    <img src="resources/Devider.png" class="divider" alt="divider">
+   
+    <footer class="footer">
+  <a href="#"><img src="resources/github.png" alt="github"></a>
+  <a href="#"><img src="resources/linkdln.png" alt="linkdln"></a>
+  <a href="#"><img src="resources/Twitter.png" alt="twitter"></a>
+  <a href="#"><img src="resources/Angellist.png" alt="angel"></a>
+  <a href="#"><img src="resources/Medium.png" alt="medium"></a>
```

Comment:
- [ ] Kindly consider adding more spaces between icons to match the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A2346)
- [ ] The list of icons uses a ul tag to group them instead of a footer, and li per icon; each icon has a link (a tag) like